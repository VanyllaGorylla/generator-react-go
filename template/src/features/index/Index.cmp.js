import React from 'react';
import { Container, Card } from 'react-materialize';

const Index = () => (
  <Container className="index">
    <Card title="Hello my friend!!" />
  </Container>
);

export default Index;
