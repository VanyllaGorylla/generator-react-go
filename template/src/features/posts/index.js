import Posts from './Posts.cmp';

import PostsEdit from './PostsEdit.cmp';

export { Posts, PostsEdit };
