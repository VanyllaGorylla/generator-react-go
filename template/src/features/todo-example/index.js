export { default as TodoExample } from './TodoExample.cmp';
