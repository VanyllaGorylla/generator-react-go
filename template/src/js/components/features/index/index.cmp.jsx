import React from 'react';

const Index = () => (
	<div className="container index">
		<h3>Hello my friend!!</h3>
	</div>
);

export default Index;
