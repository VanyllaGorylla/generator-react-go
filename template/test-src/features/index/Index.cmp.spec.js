import React from 'react';
import { Index } from '@/features/index';

import renderer from 'react-test-renderer';

describe(`Index.cmp test`, function() {
  it(`should render propr HTML`, function() {
    const tree = renderer.create(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
