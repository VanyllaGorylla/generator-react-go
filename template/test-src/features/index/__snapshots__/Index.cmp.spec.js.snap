// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index.cmp test should render propr HTML 1`] = `
<div
  className="container"
>
  <div
    className="card"
  >
    <div
      className="card-content"
    >
      <span
        className="card-title grey-text text-darken-4"
      >
        Hello my friend!!
      </span>
      <div />
    </div>
    <div
      className="card-reveal"
    >
      <span
        className="card-title grey-text text-darken-4"
      >
        Hello my friend!!
        <i
          className="material-icons right"
        >
          close
        </i>
      </span>
    </div>
  </div>
</div>
`;
